new version README template (#49)
WordPress/block-development-examples: templates/block-examples/readmes/README-EXAMPLE.tpl.md

## Start Guide

The examples in this repo require these three steps to see them in action:

1. Install dependencies
2. Generate the build

> 3. Use it in a WordPress

Reviewer: ```suggestion
3. Use it in a WordPress installation
```

Reviewer: I think we're missing a word.

The way to manage the first two steps will be different depending on the context you're using this example.

> **Note**
> At the [WIKI of this repo](https://github.com/wptrainingteam/block-development-examples/wiki) you have the documentation for the examples on this repo

### As an example inside this monorepo

> The quickest way to have access to all the examples is clone this repository and check the examples locally.

Reviewer: ```suggestion
The quickest way to have access to all the examples is clone this repository and check out the examples locally.
```

Check the [05 Development > Repo Commands > Dependencies](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#dependencies) instructions for this repo to install the dependencies of this example.

> Once the depencies are installed (a `node_modules` folder should have been created), [launch the build process](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#build-process) to get the final version of the block that can be used from WordPress. 

Reviewer: ```suggestion
Once the depencies are installed (a `node_modules` folder should have been created), [launch the build process](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#build-process) to get the final version of the block that can be used in WordPress. 
```

Check the [WordPress Local Development Environment](https://github.com/wptrainingteam/block-development-examples/wiki/05-Development#wordpress-local-development-environment) to learn how to quickly get your local WordPress installation to see this example.

### As an isolated example on your own WordPress installation

> If you have a local WordPress installation already running, you can also put the plugin folders of the examples you're interested in (by copyin and paste, from the zips...) inside the `plugins` folder of that installation and check the examples there.

Reviewer: ```suggestion
If you have a local WordPress installation already running, you can also put the plugin folders of the examples you're interested in (by copying and pasting, from the zips...) inside the `plugins` folder of that installation and check the examples there.
```
